link-user-to-episys-name-record/REPWRITERSPECS/BANNO.DATABASE.CHECK.V1.POW

```diff
@@ -765,7 +951,11 @@ PROCEDURE GETNAMETYPES
 ** second in order of name type.
 *]
  VALIDNAMETYPECOUNT=0
- LELISTINPUT=@RGUSERCHR1
+ TMPCHR=@RGUSERCHR1
+ CALL NLTS
+ LELISTINPUT=TMPCHR
+ IF LELISTINPUT="" THEN
+  LELISTINPUT="ALL"
```

Comment:
Not sure if we want ALL name types if they send us nothing. In the Banno DB though if it is blank right now they are sending us a name type list of 0,1,5-9 per what they said the default is for name types in documentation. Not sure if this should be change to "0,1,5-9" if it comes as blank, or if Leaving it as ALL is fine since as of now they will send us a specific list, or they will send us 0,1,5-9 if the Banno DB has no values.

Reply:
This was really added in as a stop-gap.  We found that there were times when Banno was sending back a null string in which case the listexpand code was interpreting that as a "none". Better to allow ALL name records instead of none.